### ISSUE TITLE
Additional mappings for metrics-conversion-prod

### ISSUE THREAD

**Issue by taceybadgerbrook** [wellcomecollection/platform#5360]
The following fields cannot be used in Kibana dashboard aggregations because they are not of type keyword. Text doesn't appear to allow counts.

|field|Used to count|
|:---|:---|
|page.query.query|queries with null results, popular queries|
|page.query.workId|popular items viewed|
|page.query.id|archive works pages viewed|

**Comment by jamesgorrie** [comment]
If we change `query` to `keyword`, would we want to keep they `text` value of it? If so, what's the use case?

**Comment by taceybadgerbrook** [comment]
I can imagine the text value being used in data science work, so I would want to keep this. I think it's better if we add a field with keyword value, eg page.query.query.keyword.

**Comment by jamesgorrie** [comment]
Happy to ensure we have both types.

It would be good to know what the use cases are to ensure we're structuring the data to our [information needs](https://en.wikipedia.org/wiki/Information_needs) to avoid us getting into the somewhat messy state of the last `conversion` index.

**Comment by taceybadgerbrook** [comment]
I agree. Conversion* had all the types all the time. Shall we wait for Harrison's view on keeping the text value and using keyword value in dashboards? I am assuming keyword type is required for counting because Kibana index management shows only keywords and numbers are aggregatable but I could be wrong.

**Comment by taceybadgerbrook** [comment]
Here's how I'm thinking this could work:

|New field|Based on existing field - to keep for data science work|
|:---|:---|
|page.query.query.keyword (type:keyword)|page.query.query (type:text)|
|page.query.workId.keyword (type:keyword)|page.query.workId (type:text)|
|page.query.id.keyword (type:keyword)|page.query.id (type:text)|
